runtime/compiler/runtime/RelocationRecord.cpp

```diff
@@ -4561,3 +4561,109 @@ TR_RelocationRecordClassUnloadAssumption::applyRelocation(TR_RelocationRuntime *
    reloTarget->addPICtoPatchPtrOnClassUnload((TR_OpaqueClassBlock *) -1, reloLocation);
    return 0;
    }
+
+
+
+uint32_t TR_RelocationRecord::_relocationRecordHeaderSizeTable[TR_NumExternalRelocationKinds] =
+   {
+   sizeof(TR_RelocationRecordConstantPoolBinaryTemplate),                            // TR_ConstantPool                                 = 0
+   sizeof(TR_RelocationRecordHelperAddressBinaryTemplate),                           // TR_HelperAddress                                = 1
+   24,                                                                               // TR_RelativeMethodAddress                        = 2
```

Comment:
I presume the plan is to eventually have these values computed via sizeof as well?

Reply:
Yeah, the reason I didn't do it in the initial commit is because there are some where the value returned by `bytesInHeaderAndPayload` doesn't match the size in the table :) so I figure I'll deal with that mismatch in the follow up cleanup where a single commit with deal with a single relo record.